src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -1,33 +1,57 @@
 .context-nav-wrapper {
   border: none !important;
+  border-top-right-radius: $euiSizeL;
+  border-bottom-right-radius: $euiSizeL;
+  background-color: $euiColorLightShade;
```

Comment:
This background color doesn't look like the color in design, shall we use the color from design?

Reply:
There is not a token for left nav in OUI yet, I'd like to merge this version first. We can raise another PR to adjust background color if we have any new updates.

Reply:
It looks the background is one of the important change and affects the look & feel quite a bit, my comments below are also related, let me double check and ask UX to confirm.

If we'd like to merge this PR first, perhaps I'd rather prefer to use the color value from design first, then then replace it with the color defined in OUI color palettes when it's ready.

Reply:
Or maybe we can have the background color change in a separate PR so that we can get the other changes in this PR merged asps.

Reply:
@virajsanghvi has a PR to add the color to OUI https://github.com/opensearch-project/oui/pull/1430/files#diff-67cb04d06a675c67602ff65a876373033b16a759964be1b850bbfb4ca50bad6aR13

And confirmed with him that we can commit the changes and let him know so that he can update the color after the PR in OUI merged. @SuZhou-Joe

Reply:
Sure, makes sense. I will use the color from design in this PR to unblock review.